corehq/apps/translations/tests/test_bulk_app_translation.py

```diff
@@ -821,6 +821,10 @@ def test_removing_form_translations(self):
             (MODULES_AND_FORMS_SHEET_NAME,
              (('Menu', 'menu1', 'Untitled Module', '', '', '', '', '', '765f110eb62fd26240a6d8bcdccca91b246b96c9'),
               ('Form', 'menu1_form1', 'Untitled Form', '', '', '', '', '', 'fffea2c32b7902a3efcb6b84c94e824820d11856'))),
+              # TODO: This unique ID ^^^ does not match the one in form_modifications/app.json. Why?
```

Comment:
There are two things about these test failures that I don't understand. The first is mismatched form `unique_id` values. Try this:
1. Pick any app in your local environment. The smaller the better. Open its JSON source in your browser.
2. Open a Django shell, and get the app. e.g.:
```python
>>> app_id = '298b67da52ecef99b055077d710034ec'  # copied from the URL
>>> from corehq.apps.app_manager.dbaccessors import get_app
>>> app = get_app('demo', app_id)
>>> for m in app.get_modules():
...      print(m.unique_id)
...      for f in m.get_forms():
...          print(f.unique_id)

57d4ea1a2d09412a94123ace6de8d79c
0368dbd58e80484eb1d2acfc16d603de
b0e6333fd78e4578a1c1e48fd047a878
```
3. Search for the module's unique_id in your browser. It finds it.
4. Now search for a form's unique_id. Mine is not found. The form's unique_id values in the browser are different from the values in the shell. Why?!